### DISCUSSION TITLE
Alert dispositivo offline

### DISCUSSION THREAD

**Discussion by Pico1965** [indomus/forum#1394] [HUB: Home Assistant]
Buonasera.

Volevo sapere se esiste un addon (o un yaml già fatto) che verifica se i dispositivi siano online e, in caso contrario, inviano una notifica.
Mi spiego... diciamo che ho tre device. Quando uno va offline vorrei ricevere una notifica del tipo "L'interruttore 1 è andato offline"
Un po' come fa l'App SmartHome

**Comment by snakuzzo** [comment]
1... si scrive "Alert"
2. Puoi farlo semplicemente con un'automazione in base allo stato dell'entità e/o dispositivo. Ma va visto in base al tipo di entità che hai creato (con quale platform più che altro)
